integration-tests/batch-api.spec.ts

```diff
@@ -29,7 +29,7 @@ CONFIGS().forEach(({ lib, rpc, setup, knownBaker, createAddress }) => {
     })
 
     it('Simple transfers with origination using with', async (done) => {
-      const op = await Tezos.batch([
+      const op = await Tezos.contract.batch([
```

Comment:
Should we keep this test as it until we removed the deprecated batch function?

contract.batch() is covered in the file contract-batch.spec.ts